filetracker/__init__.py

```diff
@@ -903,6 +902,8 @@ def put_file(self, name, filename, to_local_store=True,
 
         return versioned_name
 
+    # This is a very cool method except our server doesn't support DELETE
+    # requests.
```

Comment:
Ticket i odnośnik do niego? To jest ewidentnie brakująca funkcjonalność, którą powinniśmy mieć.

Reply:
Done.